src/main/scala/org/ergoplatform/settings/ErgoSettings.scala

```diff
@@ -109,6 +109,7 @@ object ErgoSettings extends ScorexLogging
       .defaultOverrides()
       .withFallback(cfg)
       .withFallback(firstFallBack)
+      .withFallback(ConfigFactory.defaultApplication())
```

Comment:
having `reference.conf` on ergo's classpath with  `scorex {` scope conflicts with `reference.conf` from scorex framework ... typesafe config merges configs from these files sort of as classloader loads it from the classpath (non-deterministically, it depends on underlying JVM type) ... I guess if we want ergo's scorex configs to override scorex's scorex configs, we should use `application.conf`  instead